### DISCUSSION TITLE
Remove/archive some old plugin repos

### DISCUSSION THREAD

**Discussion by qwerty287** [woodpecker-ci/woodpecker#2848] [General]
@woodpecker-ci/owner 

I'm talking about these repos:
- https://github.com/woodpecker-ci/plugin-email (there's a 3rd-party email plugin and this one is just a fork of the drone plugin without any changes)
- https://github.com/woodpecker-ci/plugin-chart-releaser (just a WIP repo with only a README, but stale for >1.5 years)
- https://github.com/woodpecker-ci/plugin-base (fork of drone without changes)
- https://github.com/woodpecker-ci/plugin-git-2 (we have a working plugin-git and this one is just fork of drone without changes)
- https://github.com/woodpecker-ci/plugin-kaniko (we have a working docker-buildx and this one is just fork of drone without changes)
- https://codeberg.org/woodpecker-plugins/plugin-base (private, but only contains a README)
- https://codeberg.org/woodpecker-plugins/bootstrap-woodpecker-plugin (private, but only contains a README)

As these repos never got any additional contributions and they're also not published on docker or similar, I think to clean up our repos we should delete them. If you think deleting is bad also just archiving would be nice.

**Comment by anbraten** [comment]
Will try to update / get working:

- https://github.com/woodpecker-ci/plugin-chart-releaser
- https://github.com/woodpecker-ci/plugin-kaniko

The rest can be archived from my side.

**Reply by qwerty287** [reply]
@anbraten 

Looks like still nothing happened for these plugins from our side, but somebody opened a PR on the kaniko plugin few days ago: https://github.com/woodpecker-ci/plugin-kaniko/pull/1

**Comment by 6543** [comment]
If there is no change and fork origin exists ... i'm going to delete it, else archive them should be fine

**Reply by qwerty287** [reply]
Thanks!

What about these:

- https://github.com/woodpecker-ci/plugin-base (fork of drone without changes)
- https://codeberg.org/woodpecker-plugins/plugin-base (private, but only contains a README)
- https://codeberg.org/woodpecker-plugins/bootstrap-woodpecker-plugin (private, but only contains a README)

**Reply by qwerty287** [reply]
And I also just saw https://github.com/woodpecker-ci/envsubst which is unused as well.

**Reply by anbraten** [reply]
did some cleanup

**Reply by qwerty287** [reply]
Nice! I'll close this as all repos are done now.
